docs/v.list

```diff
@@ -20,13 +20,13 @@
 
     <var name="minGradleVersion" value="6.8.3" type="string"/>
 
-    <var name="maxGradleVersion" value="7.3.3" type="string"/>
+    <var name="maxGradleVersion" value="7.6" type="string"/>
```

Comment:
Gradle has already released 7.6.1, probably makes sense to mention it instead of 7.6
```suggestion
    <var name="maxGradleVersion" value="7.6.1" type="string"/>
```
(and the same for `gradleVersion`)

Reply:
I usually take the version from [TestVersions.kt](https://github.com/JetBrains/kotlin/blob/1.8.20/libraries/tools/kotlin-gradle-plugin-integration-tests/src/test/kotlin/org/jetbrains/kotlin/gradle/testbase/TestVersions.kt). Can we be sure about 7.6.1 if in TestVersions it's 7.6? Was it tested?

Reply:
`TestVersions.kt` should be updated as well

Reply:
IDK if it's possible having the release in two days? I know this change is minor, but I'm not familiar with the processes of developing Kotlin itself.

Reply:
@Tapchicoma WDYT? Can we just update the version? I've started the remote run for `master`, but not the release branch

Reply:
Ok, I'm fine with it.